aspnetcore/performance/caching/middleware.md

```diff
@@ -2,49 +2,38 @@
 title: Response Caching Middleware in ASP.NET Core
 author: guardrex
 description: Learn how to configure and use Response Caching Middleware in ASP.NET Core.
-monikerRange: '>= aspnetcore-1.1'
+monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 02/20/2019
+ms.date: 07/05/2019
 uid: performance/caching/middleware
 ---
 # Response Caching Middleware in ASP.NET Core
 
 By [Luke Latham](https://github.com/guardrex) and [John Luo](https://github.com/JunTaoLuo)
 
-[View or download sample code](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/performance/caching/middleware/samples) ([how to download](xref:index#how-to-download-a-sample)).
+[View or download sample code](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/performance/caching/middleware/samples) ([how to download](xref:index#how-to-download-a-sample))
 
-This article explains how to configure Response Caching Middleware in an ASP.NET Core app. The middleware determines when responses are cacheable, stores responses, and serves responses from cache. For an introduction to HTTP caching and the `ResponseCache` attribute, see [Response Caching](xref:performance/caching/response).
+This article explains how to configure Response Caching Middleware in an ASP.NET Core app. The middleware determines when responses are cacheable, stores responses, and serves responses from cache. For an introduction to HTTP caching and the <xref:Microsoft.AspNetCore.Mvc.ResponseCacheAttribute> attribute, see [Response Caching](xref:performance/caching/response).
 
-## Package
-
-::: moniker range=">= aspnetcore-2.1"
-
-Reference the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app) or add a package reference to the [Microsoft.AspNetCore.ResponseCaching](https://www.nuget.org/packages/Microsoft.AspNetCore.ResponseCaching/) package.
-
-::: moniker-end
-
-::: moniker range="= aspnetcore-2.0"
-
-Reference the [Microsoft.AspNetCore.All metapackage](xref:fundamentals/metapackage) or add a package reference to the [Microsoft.AspNetCore.ResponseCaching](https://www.nuget.org/packages/Microsoft.AspNetCore.ResponseCaching/) package.
-
-::: moniker-end
+## Configuration
 
-::: moniker range="= aspnetcore-1.1"
+Use the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app) or add a package reference to the [Microsoft.AspNetCore.ResponseCaching](https://www.nuget.org/packages/Microsoft.AspNetCore.ResponseCaching/) package.
```

Comment:
The metapackage won't be used for 3.0+. Do you have another issue to address this?

Reply:
Not addressing it on this little pass. I only have another ... er ... maybe four samps to hit for 2.2.

WRT the `App` package text changing, I'll take care of it for my 3.0 pass on all of my authored topics later. This pass is 2.2 only.

Reply:
Issue is the same ... last column of #5495 ... I guessed we'd use the same process as has been used in prior releases.

However, guidance might be shifting: It seems like extra work to open one separate issue for every topic. I can just use #5495. **_But_...** if the team is judged on the **_number of issues_** resolved in a given period, then that's probably the right thing to do. I figured that we're far enough off from 3.0 at this time that I can get a final answer on that whenever. It doesn't seem critical at the moment.